erpnext/controllers/status_updater.py

```diff
@@ -13,15 +13,17 @@ def validate_status(status, options):
 
 status_map = {
 	"Lead": [
-		["Converted", "has_customer"],
+		["Lost Quotation", "lost_quotation"],
 		["Opportunity", "has_opportunity"],
+		["Quotation", "has_quotation"],
+		["Converted", "has_customer"],
```

Comment:
`status_map` gets reversed later on - so converted is evaluated first.

This could be changed/removed to help with readability - priority is set on order instead of reverse order.